composer.json

```diff
@@ -5,28 +5,33 @@
     "description": "Click and Collect plugin for Sylius, to sell and deliver securely during the COVID-19 pandemic.",
     "license": "AGPL-3.0-or-later",
     "require": {
-        "php": "^7.4",
+        "php": ">=7.4",
         "simshaun/recurr": "^4.0",
+        "sylius/calendar": "^0.3.0",
         "sylius/sylius": "^1.7",
         "symfony/lock": "^4.4 || ^5.0"
```

Comment:
Could we also add support for Symfony 6, for all SF dependencies (could be done in another PR)?